src/coreclr/jit/gentree.h

```diff
@@ -4275,6 +4275,13 @@ struct ReturnTypeDesc
 private:
     var_types m_regType[MAX_RET_REG_COUNT];
 
+#if defined(TARGET_RISCV64) || defined(TARGET_LOONGARCH64)
+    // Structs according to hardware floating-point calling convention are passed as two logical fields, each in
+    // separate register, disregarding struct layout such as packing, custom alignment, padding with empty structs, etc.
+    // We need size (can be derived from m_regType) & offset of each field for memory load/stores
+    unsigned m_fieldOffset[MAX_RET_REG_COUNT];
+#endif
+
```

Comment:
I think we should look to unify this in .NET 10 and just store the offsets for all targets. It will also remove some ifdef soup that exists for Swift handling.

Reply:
For now I think it's good to keep everything ifdeffed out since we are going into a period where we don't want to take risky changes.